sdk/lib/web3.js

```diff
@@ -136,20 +129,58 @@ async function singleCall({ web3, limiter, target, abi, block, params }) {
   return { callCount: 1, output: result };
```

Comment:
singleCall output also need to be in shape of { input, success, output }

Reply:
https://github.com/certikfoundation/TVL-Adapters/blob/master/docs/sdk.md#callobject

According to the sdk doc, the output field should only be output